src/specific/s_input.c

```diff
@@ -165,16 +176,170 @@ static INPUT_SCANCODE m_Layout[INPUT_LAYOUT_NUMBER_OF][INPUT_ROLE_NUMBER_OF] = {
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_FPS
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_BILINEAR
+    },
+
+    {
+    { BUTTON, {SDL_CONTROLLER_BUTTON_DPAD_UP}, 0 },        // INPUT_ROLE_UP
+    { BUTTON, {SDL_CONTROLLER_BUTTON_DPAD_DOWN}, 0 },      // INPUT_ROLE_DOWN
+    { BUTTON, {SDL_CONTROLLER_BUTTON_DPAD_LEFT}, 0 },      // INPUT_ROLE_LEFT
+    { BUTTON, {SDL_CONTROLLER_BUTTON_DPAD_RIGHT}, 0 },     // INPUT_ROLE_RIGHT
+    { AXIS,   {SDL_CONTROLLER_AXIS_TRIGGERLEFT}, 1 },      // INPUT_ROLE_STEP_L
+    { AXIS,   {SDL_CONTROLLER_AXIS_TRIGGERRIGHT}, 1 },     // INPUT_ROLE_STEP_R
+    { BUTTON, {SDL_CONTROLLER_BUTTON_RIGHTSHOULDER}, 0 },  // INPUT_ROLE_SLOW
+    { BUTTON, {SDL_CONTROLLER_BUTTON_X}, 0 },              // INPUT_ROLE_JUMP
+    { BUTTON, {SDL_CONTROLLER_BUTTON_A}, 0 },              // INPUT_ROLE_ACTION
+    { BUTTON, {SDL_CONTROLLER_BUTTON_Y}, 0 },              // INPUT_ROLE_DRAW
+    { BUTTON, {SDL_CONTROLLER_BUTTON_LEFTSHOULDER}, 0 },   // INPUT_ROLE_LOOK
+    { BUTTON, {SDL_CONTROLLER_BUTTON_B}, 0 },              // INPUT_ROLE_ROLL
+    { BUTTON, {SDL_CONTROLLER_BUTTON_BACK}, 0 },           // INPUT_ROLE_OPTION
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_FLY_CHEAT,
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_ITEM_CHEAT,
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_LEVEL_SKIP_CHEAT,
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_TURBO_CHEAT,
+    { BUTTON, {SDL_CONTROLLER_BUTTON_START}, 0 },          // INPUT_ROLE_PAUSE,
+    { AXIS,   {SDL_CONTROLLER_AXIS_RIGHTY}, -1 },          // INPUT_ROLE_CAMERA_UP
+    { AXIS,   {SDL_CONTROLLER_AXIS_RIGHTY}, 1 },           // INPUT_ROLE_CAMERA_DOWN
+    { AXIS,   {SDL_CONTROLLER_AXIS_RIGHTX}, -1 },          // INPUT_ROLE_CAMERA_LEFT
+    { AXIS,   {SDL_CONTROLLER_AXIS_RIGHTX}, 1 },           // INPUT_ROLE_CAMERA_RIGHT
+    { BUTTON, {SDL_CONTROLLER_BUTTON_RIGHTSTICK}, 0 },     // INPUT_ROLE_CAMERA_RESET
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_EQUIP_PISTOLS
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_EQUIP_SHOTGUN
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_EQUIP_MAGNUMS
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_EQUIP_UZIS
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_USE_SMALL_MEDI
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_USE_BIG_MEDI
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_SAVE
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_LOAD
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_FPS
+    { BUTTON, {SDL_CONTROLLER_BUTTON_INVALID}, 0 },        // INPUT_ROLE_BILINEAR
+    }
+        // clang-format on
+    };
```

Comment:
Clang format kept doing this to me for some reason. But I'll look into it.

Reply:
Seems like the indent happens because
```
static CONTROLLER_MAP
    m_ControllerLayout[INPUT_LAYOUT_NUMBER_OF][INPUT_ROLE_NUMBER_OF] = {
        // clang-format off
```
the `m_ControllerLayout` is already indented onto a second line. If I shorten this line, `// clang-format off` does not get indented. Not sure there's a way around this without shortening variable names.

Reply:
In that case, can we indent the inner block to match the outer indentation level?